### DISCUSSION TITLE
Is there a way to edit a value in devtools?

### DISCUSSION THREAD

**Discussion by npearson72** [jotaijs/jotai-devtools#112] [Q&A] [answered]
There are cases where I would like to modify state to test UI behavior, but I don't see how in devtools. Is there a way and I'm just missing it?

**Comment by arjunvegda** [comment, marked as answer]
Hey @npearson72 

DevTools currently is read-only and does not support modifying atom values atm

**Reply by npearson72** [reply]
Understood. Do you know if there are any plans to offer that enhancement, or is there a security issue that prevents it?

**Reply by arjunvegda** [reply]
Sorry, no plans to offer editing capabilities atm. The editing feature would bring in a lot of complexity and would be hard to cover all the use cases reliably 😅

**Reply by dai-shi** [reply]
fwiw, `useAtomDevtools` supports modifying.
https://github.com/jotaijs/jotai-devtools/blob/12ba74a21968029ec19cfaa8d81f75ddb4cb51aa/src/utils/useAtomDevtools.ts#L35
